tests/Entity/UserTest.php

```
@@ -0,0 +1,256 @@
+    public function testUserCanUpdateProfile(): void
+    {
+        $user = $this->createValidUser();
+        $user->updateDisplayName('New Display Name');
+
+        self::assertSame('New Display Name', $user->getDisplayName());
+    }
+
+    public function testUserCanUpdateUsername(): void
+    {
+        $user = $this->createValidUser();
+        $originalUsername = $user->getUsername();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$originalUsername'. (undefined)

(UnusedLocalVariable)